docs/user-guide/parameters.md

```diff
@@ -146,7 +146,7 @@ A csv file that describes the routing of flow to the edge of the origin grid cel
     - Type: bool
     - Valid values: True, False
 
-    Note: *True when routing to coastal grid cells, else False.*
+    *Default: True*
```

Comment:
There is no default value here. Users will have to specify one way or another. I think we want to say something like:

```
Note: In most cases, this should be set to True unless routing is being done to nested basins, in which case, this should be set to False.
```